lib/globals/linterRules.sh

```
@@ -19,7 +19,7 @@ CLOUDFORMATION_FILE_NAME=".cfnlintrc.yml"
 COFFEESCRIPT_FILE_NAME=".coffee-lint.json"
 CSS_FILE_NAME="${CSS_FILE_NAME:-.stylelintrc.json}"
 DOCKERFILE_HADOLINT_FILE_NAME="${DOCKERFILE_HADOLINT_FILE_NAME:-.hadolint.yaml}"
-EDITORCONFIG_FILE_NAME="${EDITORCONFIG_FILE_NAME:-.ecrc}"
```

As this is implemented, it's a breaking change. Let me think if how we can make it a non-breaking change, or at least require explicit user action.

---

Otherwise, users that rely on the old value being set, will see their editorconfig-checker configuration silently ignored